plugins/usi-dock/fu-plugin-usi-dock.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2021 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <fwupdplugin.h>
+
+#include "fu-usi-dock-dmc-device.h"
+#include "fu-usi-dock-firmware.h"
+#include "fu-usi-dock-mcu-device.h"
+
+#define USI_DOCK_TBT_INSTANCE_ID "THUNDERBOLT\\VEN_0108&DEV_2031"
```

Comment:
just to confirm since this used plural of docks - is this the only TBT ID handled by the plugin internally for now?  Need to make sure that these stay in sync as more docks are released.

Reply:
for the present, yes.